.github/workflows/devtools.yml

```diff
@@ -125,11 +125,7 @@ jobs:
       # e.g. if set 1 fails, continue with set 2 anyway
       fail-fast: false
     runs-on: macos-latest
-    env:
-      # The total number of shards. Set dynamically when length of single matrix variable is
-      # computable. See https://github.community/t/get-length-of-strategy-matrix-or-get-all-matrix-options/18342
-      SHARD_TOTAL: 2
-    name: Smoke batch ${{ matrix.smoke-test-shard }}
+    name: Smoke DevTools ${{ matrix.smoke-test-shard }}
```

Comment:
nit: "DevTools smoke" or change the ordering of the other names (Ex: "Smoke windows")

Reply:
I considered that, but then we end up with `DevTools / DevTools smoke` or `smoke / Smoke Windows` haha. I guess it doesn't really matter and consistency is better :)